### PR TITLE
Performance Changes

### PR DESCRIPTION
**Pull request by okbel**
- Removing HeadlessUI
- Removing React-Aria

Both of them adds significant amount of kb to the final bundle.

Solves:
https://github.com/vercel/commerce/issues/87
https://github.com/vercel/commerce/issues/88

### CODE REVIEW

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/wishlist**](https://commerce-git-perf.vercel.vercel.app/wishlist) |
|:-:|
|<a href="https://commerce-git-perf.vercel.vercel.app/wishlist">Screenshot of /wishlist</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/63Z3KN_axz7OTyKij3LFe_kfyjs_.png">(view full size)</a> |







Commit 58067808d7e56564843ea5cda1ac5352b6badaf8 (https://commerce-git-perf.vercel.vercel.app).

**Comment by okbel** [comment]
@marbiano Take a loo. LMK what you think. We need to check a11y.

File: components/common/I18nWidget/I18nWidget.tsx [hunk 1]
```diff
@@ -32,49 +30,69 @@ const LOCALES_MAP: Record<string, LOCALE_DATA> = {
...
+              >
+                <Cross className="h-6 w-6" />
+              </button>
+            </div>
+            <ul>
+              {options.map((locale) => (
+                <li key={locale}>
                   <Link href={currentPath} locale={locale}>
-                    <a className={cn(s.item, { [s.active]: active })}>
+                    <a className={cn(s.item)} onClick={() => setDisplay(false)}>
```
Comment (marbiano) [inline components/common/I18nWidget/I18nWidget.tsx:86]: Why is that state and `onClick` event needed?

File: components/common/I18nWidget/I18nWidget.tsx [hunk 2]
```diff
@@ -32,49 +30,69 @@ const LOCALES_MAP: Record<string, LOCALE_DATA> = {
...
-                {({ active }) => (
+      <div className="flex items-center relative">
+        <button className={s.button} aria-label="Language selector" />
+        <img
+          className="block mr-2 w-5"
+          src={`/${LOCALES_MAP[currentLocale].img.filename}`}
+          alt={LOCALES_MAP[currentLocale].img.alt}
+        />
+        {options && (
+          <span className="cursor-pointer" onClick={() => setDisplay(!display)}>
```
Comment (marbiano) [inline components/common/I18nWidget/I18nWidget.tsx:53]: Are `@component/icons` also gone?

File: components/common/UserNav/DropdownMenu.tsx [hunk 3]
```diff
@@ -29,68 +29,70 @@ const LINKS = [
...
-  const { theme, setTheme } = useTheme()
   const logout = useLogout()
   const { pathname } = useRouter()
-
+  const { theme, setTheme } = useTheme()
+  const [display, setDisplay] = useState(false)
   const { closeSidebarIfPresent } = useUI()
 
   return (
-    <Transition
```
Comment (marbiano) [inline components/common/UserNav/DropdownMenu.tsx:39]: Are transitions gone?

File: package.json [hunk 4]
```diff
@@ -43,25 +43,25 @@
...
     "@reach/portal": "^0.11.2",
-    "@react-aria/overlays": "^3.4.0",
     "@tailwindcss/ui": "^0.6.2",
+    "@types/body-scroll-lock": "^2.6.1",
     "@types/lodash.throttle": "^4.1.6",
     "@vercel/fetch": "^6.1.0",
+    "body-scroll-lock": "^3.1.5",
     "bowser": "^2.11.0",
     "classnames": "^2.2.6",
+    "contentstack": "^3.11.0",
```
Comment (marbiano) [inline package.json:54]: Why was this dependency added?

File: pages/wishlist.tsx [hunk 5]
```diff
@@ -15,62 +15,35 @@ export async function getStaticProps({
...
-    props: { pages },
+    props: { ...defatultPageProps, pages },
   }
 }
 
 export default function Wishlist() {
   const { data, isEmpty } = useWishlist({ includeProducts: true })
 
   return (
-    <Container>
```
Comment (marbiano) [inline pages/wishlist.tsx:26]: Was `Container` removed?

**Review by marbiano — COMMENTED** [review]
@okbel great work! A couple of questions around, but in general looks like amazing work. 

Seems to me like two important actionables right after this PR would be to work on accessibility now that react-aria is gone, and transitions with the headless UI library also gone, right?

File: components/common/I18nWidget/I18nWidget.tsx (hunk 1 above)
Comment (okbel) [reply components/common/I18nWidget/I18nWidget.tsx:86]: We removed Menu Component from headless UI. We need to close it some how.

File: components/common/I18nWidget/I18nWidget.tsx (hunk 2 above)
Comment (okbel) [reply components/common/I18nWidget/I18nWidget.tsx:53]: No

File: components/common/UserNav/DropdownMenu.tsx (hunk 3 above)
Comment (okbel) [reply components/common/UserNav/DropdownMenu.tsx:39]: They are from headlessUI

File: package.json (hunk 4 above)
Comment (okbel) [reply package.json:54]: Merge issue. Will remove

File: pages/wishlist.tsx (hunk 5 above)
Comment (okbel) [reply pages/wishlist.tsx:26]: No

**Pull request merged**